packages/private-build-infra/addon/current-deprecations.ts

```diff
@@ -51,4 +51,5 @@ export default {
   DEPRECATE_SERIALIZER_QUERY_RECORD_ARRAY_RESPONSE: '3.4',
   DEPRECATE_BELONGS_TO_REFERENCE_PUSH: '3.16',
   DEPRECATE_REFERENCE_INTERNAL_MODEL: '3.19',
+  DEPRECATE_NAJAX: '4.0',
```

Comment:
We should maybe clean up the description here, the version here is not "in what version will this deprecation be automatically removed" but rather "in what version was this deprecation introduced", in the case of current master that would be 3.21. That way an application can be on say version 3.25 and maybe not have resolved a deprecation introduced in 3.23 but can still strip resolved deprecations from versions prior by specifying 3.22